README.md

```diff
@@ -48,24 +71,82 @@ Then update to your own private key.
 
 ## Print wallet balances
 
-This script will print your wallet balances for each chain.
+To print your wallet balances for each chain, run:
 
 ```bash
 npm run check-balance [local|testnet]
 ```
 
-If not specified, this will print balances of the wallet for testnet.
+The aforementioned command pertains to specifying the intended environment for a project to execute on. It provides the option to choose between local and testnet environments by appending either `local` or `testnet` after the command. 
```

Comment:
I don't think this is necessary here because we actually have a default. This is also very wordy, we could just say "`npm run check-balance local` will print balances for the locally running blockchains" or similar.

Reply:
Updated